docs/modeling/bounding-boxes.rst

```diff
@@ -0,0 +1,216 @@
+
+        >>> g2.bounding_box = [-4.5, 4.5, -3.0, 3.0]
+        >>> g2.bounding_box
+        ((-4.5, 4.5), (-3.0, 3.0))
+
+The default is reset with ``'auto'``.
+
+        >>> g2.bounding_box = 'auto'
+        >>> g2.bounding_box
+        ((-11.0, 11.0), (-5.5, 5.5))
+
+The ``bounding_box`` is turned off with `None`.
+
+        >>> g2.bounding_box = None
+        >>> g2.bounding_box
+        >>> 
+
+The default box for Gaussian models is :math:`\pm 5.5 \sigma` from the mean, 
+but this factor can be passed to ``bounding_box_default`` for the ``Gaussian1D`` 
+and ``Gaussian2D`` models.
+
+        >>> g1.stddev = 3
+        >>> g1.bounding_box = g1.bounding_box_default(factor=10)
+        >>> g1.bounding_box
+        (-30.0, 30.0)
+
+The default function can also be set to any callable. This is particularly 
+useful for fitting ``custom_model`` or ``CompoundModel`` instances. 
+
+
+        >>> from astropy.modeling import custom_model
+        >>> def sphere(x, y, z, x0=0, y0=0, z0=0, r0=5, amp=1):
+        ...     rsq = x ** 2 + y ** 2 + z ** 2
+        ...     val = (rsq < r0 ** 2) * amp
+        ...     return val
+        ...
+        >>> def sphere_bb(self):
+        ...     return ((self.x0 - self.r0, self.x0 + self.r0),
+        ...             (self.y0 - self.r0, self.y0 + self.r0),
+        ...             (self.z0 - self.r0, self.z0 + self.r0))
+        ...
+        >>> Sphere3D = custom_model(sphere)
+        >>> Sphere3D.bounding_box_default = sphere_bb
+        >>> model = Sphere3D()
+        >>> model.bounding_box = 'auto'
+        >>> model.bounding_box
+        ((-5.0, 5.0), (-5.0, 5.0), (-5.0, 5.0))
+
+Efficient evaluation with ``render_model``
+------------------------------------------
+
+When a model is evaluated over a range much larger than the model itself, it may 
+be prudent to use `astropy.modeling.render_model` if efficiency is a concern. 
+The ``render_model`` function can be used to evaluate a model on an array of the 
+same dimensions. If no array is given, ``render_model`` will return a "postage 
+stamp" array corresponding to the bounding box region. However, if 
+``bounding_box`` is `None` an image or coordinates must be passed. To update the 
+input array in place, set ``copy = False``.
+
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        from astropy.modeling import models, render_model
+
+        imshape = (450, 600)
+        nsrc = 50
+        np.random.seed(0)
+
+        # Generate a random list of models:
+
+       model_params = [dict(amplitude=np.abs(np.random.uniform(0, 10)),
+                        x_mean = np.random.uniform(0, imshape[1]),
+                        y_mean = np.random.uniform(0, imshape[0]),
+                        x_stddev = np.abs(np.random.uniform(3, 6)),
+                        y_stddev = np.abs(np.random.uniform(3, 6)),
+                        theta = np.random.uniform(0, 2 * np.pi)) for i in range(nsrc)]
+
```

Comment:
@adonath Fixed. I had done this for the other example below but forgot this one would need the same changes. I also removed the example from the `render_model` docstring like we discussed.